src/openfermion/resource_estimates/pbc/df/compute_lambda_df.py

```diff
@@ -0,0 +1,106 @@
+# coverage: ignore
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+from dataclasses import dataclass
+import numpy as np
+import numpy.typing as npt
+
+from openfermion.resource_estimates.pbc.df.df_integrals import (
+    DFABKpointIntegrals,)
+from openfermion.resource_estimates.pbc.hamiltonian import (
+    HamiltonianProperties,)
+
+
+@dataclass
+class DFHamiltonianProperties(HamiltonianProperties):
+    """Light container to store return values of compute_lambda function"""
```

Comment:
I'd remove "light container" -- redundant. You might want to consider adding a note that this extends the parameters in `HamiltonianProperties` and isn't literally one integer in a dataclass 😂 

wouldn't hurt to document `num_eig` either